<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tartarugas fofas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 205, 0);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>DADOS</string>
     </attribute>
     <widget class="QTableWidget" name="Tabela">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>631</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 87, 83);
background-color: rgb(252, 233, 79);</string>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>104</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>NINHO</string>
       </property>
       <property name="font">
        <font>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="background">
        <color>
         <red>252</red>
         <green>233</green>
         <blue>79</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PRAIA</string>
       </property>
       <property name="background">
        <color>
         <red>252</red>
         <green>233</green>
         <blue>79</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DESOVA</string>
       </property>
       <property name="background">
        <color>
         <red>252</red>
         <green>233</green>
         <blue>79</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SINALIZAÇÃO</string>
       </property>
       <property name="background">
        <color>
         <red>252</red>
         <green>233</green>
         <blue>79</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NASCIMENTO</string>
       </property>
       <property name="background">
        <color>
         <red>252</red>
         <green>233</green>
         <blue>79</blue>
        </color>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 87, 83);
background-color: rgb(252, 233, 79);</string>
      </property>
      <property name="text">
       <string>CADASTRAR NINHO</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>211</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="test">
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 87, 83);</string>
           </property>
           <property name="text">
            <string>PRAIA:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cB_1">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(252, 233, 79);
color: rgb(85, 87, 83);</string>
           </property>
           <item>
            <property name="text">
             <string>BESSA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MANAÍRA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CABO BRANCO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TAMBAÚ</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SEIXAS</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PENHA</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 87, 83);</string>
           </property>
           <property name="text">
            <string>DESOVA:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="data_1">
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 87, 83);
background-color: rgb(252, 233, 79);</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>2019</year>
             <month>12</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 87, 83);</string>
           </property>
           <property name="text">
            <string>SINALIZAÇÃO:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cB_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(252, 233, 79);
color: rgb(85, 87, 83);</string>
           </property>
           <item>
            <property name="text">
             <string>PENDENTE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FINALIZADO</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>21</width>
        <height>331</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/WhatsApp Image 2019-12-06 at 14.16.21(1).jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>ESTATÍSTICAS</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="txt_Manaira">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>Manaíra: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txt_QuantNinhosManaira">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="txt_CaboBranco">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>Cabo Branco: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txt_QuantNinhosCaboBranco">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="txt_Tambau">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>Tambaú: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txt_QuantNinhosTambau">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="txt_Seixas">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>Seixas: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txt_QuantNinhosSeixas">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="txt_Penha">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>Penha:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txt_QuantNinhosPenha">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="txt_Bessa">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>Bessa: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txt_QuantNinhosBessa">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>QUANTIDADE DE NINHOS CADASTRADOS:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="quantNinhos">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/WhatsApp Image 2019-12-06 at 14.16.21(1).jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuARQUIVO">
    <property name="title">
     <string>ARQUIVO</string>
    </property>
    <addaction name="actionSALVAR"/>
    <addaction name="actionCARREGAR"/>
   </widget>
   <addaction name="menuARQUIVO"/>
  </widget>
  <action name="actionSALVAR">
   <property name="text">
    <string>SALVAR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCARREGAR">
   <property name="text">
    <string>CARREGAR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
